index.html

```diff
@@ -386,11 +386,139 @@ <h2 id="protocol-registry">
       the future.
     </p>
     <h3>
-      Inclusion criteria
+      General Inclusion criteria
+    </h3>
+    <aside class="note">
+      The below criteria are a work in progress and are likely to change as
+      this document evolves.
+    </aside>
+    <p>
+      To be included in the registry, the [=digital credential/exchange
+      protocol=]:
+    </p>
+    <ol>
+      <li>MUST be standardized at a <a href="https://www.w3.org/liaisons/">consortium the W3C liaises with</a>
+      <li>
+        MUST have a stable URL that points to a publicly available specification.
+      </li>
+      </li>
+      <li>MUST define a [[WebIDL]] [=dictionary=] representation, or JSON object representation, of the
+      [=digital credential/exchange protocol=] request structure (i.e., the
+      [=dictionary=] to which the {{DigitalCredentialsProvider}}'s
+      {{DigitalCredentialsProvider/request}} is [=converted to idl
+      values|converted=] to before it is passed onto underlying platform).
+      </li>
+      <li>MUST define a [[WebIDL]] [=dictionary=] representation, or JSON object representation, of the
+      [=digital credential/exchange protocol=] response structure (i.e., the
+      [=dictionary=] to which the {{DigitalCredential}}'s
+      {{DigitalCredential/data}} is [=converted to idl values|converted=] before it
+      is made available to the relying party).
+      </li>
+      <li>MUST define validation rules for members of the request and response
+      structures.
+      </li>
+      <li>MUST have undergone privacy review by the W3C's Privacy Interest
+      Group and Federated Identity Working Group.
+      </li>
+      <li>MUST have undergone security review by the Federated Identity Working
+      Group.
+      </li>
+      <li>MUST have implementation commitment from at least two implementers in
+      independent browser engines, to meet the W3C's adequate implementation
+      experience requirements.
```

Comment:
I'm not sure this makes sense. What examples of other W3C registries do we have and how they translate W3C interoperability requirements down to the things they point at?

Reply:
In particular, the primary implementation of these protocols is not browsers, it's wallets and verifiers. Therefore according to [W3C process](https://www.w3.org/policies/process/#implementation-experience) I believe it's sufficient to just have multiple wallets supporting a protocol.

Reply:
Discussed during the hybrid meeting on 2025-04-11 and there was loose consensus to remove this and re-evaluate in the future if needed.